scripts/release/release-utils.js

```diff
@@ -94,21 +94,48 @@ function updatePackageSwift (packageSwiftContent, version) {
 /**
  * Provide the source Package.resolved for BSK and get it updated with the autofill version
  * @param {string} packageResolvedContent
- * @param {string} version
- * @param {string} commit
+ * @param {{
+ *     autofill: {
+ *         version: string,
+ *         commit: string
+ *     },
+ *     bsk?: {
+ *         commit: string
+ *     }
+ * }} substitutions
  * @returns {string}
  */
-function updatePackageResolved (packageResolvedContent, version, commit) {
-    const autofillPackageResolvedRegex = new RegExp(
-        /("package": "Autofill",\s+"repositoryURL": "https:\/\/github.com\/duckduckgo\/duckduckgo-autofill.git",\s+"state": {\s+"branch": null,\s+"revision": ")(\w+)(",\s+"version": ")([\d.]+)("\s+})/,
-        'i'
-    )
+function updatePackageResolved (packageResolvedContent, substitutions) {
+    const regexes = {
+        autofill: new RegExp(
+            /("location" : "https:\/\/github.com\/duckduckgo\/duckduckgo-autofill.git",\s+"state" : {\s+"revision" : ")(\w+)(",\s+"version" : ")([\d.]+)("\s+})/,
+            'i'
+        ),
+        bsk: new RegExp(
+            /("location" : "https:\/\/github.com\/duckduckgo\/BrowserServicesKit",\s+"state" : {\s+"revision" : ")(\w+)(",\s+"version" : "[\d.]+)("\s+})/,
+            'i'
+        )
+    }
```

Comment:
The replacement string and regex are tightly coupled (by necessity) given we're using the group index for replacement. It feels like it would make sense to keep them co-located in the code. Perhaps we should move each regex into a variable within each condition block?